lib/console_helpers.rb

```diff
@@ -17,4 +17,20 @@ def inspect_submission(submission)
       STDERR.puts "\n"
     end
   end
+
+  # Outputs email addresses for users that have outstanding tasks for the given
+  # month and year, including the Frameworks they are overdue with.
+  def naughty_list(month:, year:)
+    late_tasks_scope = Task.where(period_month: month, period_year: year).where.not(status: :completed)
+    suppliers = Supplier.joins(:tasks).merge(late_tasks_scope)
+
+    suppliers.each do |supplier|
+      late_frameworks = supplier.tasks.merge(late_tasks_scope).map(&:framework)
```

Comment:
What's the order of magnitude for how many of these there are? I'm trying to preempt any slow queries or array operations, but if it's going to be a small number I suppose it doesn't matter much.